validation/citrus-validation-xml/src/main/java/org/citrusframework/validation/xml/schema/XmlSchemaValidation.java

```diff
@@ -33,14 +24,45 @@
 import org.w3c.dom.Document;
 import org.xml.sax.SAXParseException;
 
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static java.lang.Boolean.parseBoolean;
+import static java.util.Optional.ofNullable;
+
 public class XmlSchemaValidation implements SchemaValidator<XmlMessageValidationContext> {
 
+    public static final String FAIL_IF_NO_SCHEMA_FOUND_PROPERTY_NAME = "org.citrusframework.xml.fail.if.no.schema.found";
```

Comment:
I think `citrus.xml` is enough, because we've other properties that start this way. e.g. `citrus.yaml` or `citrus.json` etc.

how about `citrus.xml.no.schema.found.strategy`? that could be an `enum` with the values `IGNORE` and `FAIL`. it leaves the option to add other behaviors in the future, without breaking much.